hazelcast/src/main/java/com/hazelcast/internal/util/AddressUtil.java

```diff
@@ -364,7 +365,7 @@ public static AddressMatcher getAddressMatcher(String address) {
         return selectedPorts;
     }
 
-    public static Collection<Address> getAliases(InetSocketAddress inetSocketAddress) {
+    public static Collection<Address> getAliases(@Nonnull InetSocketAddress inetSocketAddress) {
```

Comment:
I'm curious if there is a reason to require non `null`s. Maybe we could allow it and return an empty collection instead. WDYT?

Reply:
There is not any specific reason, I just want to handle it in this way. I can change it.